Index: webmini2/src/main/java/com/sparta/webmini2/dto/SignupRequestDto.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.sparta.webmini2.dto;\r\n\r\nimport lombok.Getter;\r\nimport lombok.Setter;\r\n\r\n@Setter\r\n@Getter\r\npublic class SignupRequestDto {\r\n    private String username;\r\n    private String password;\r\n    private String passwordCheck;\r\n    private String nickName;\r\n    private String position;\r\n\r\n    public SignupRequestDto(String username, String password, String passwordCheck, String nickName, String position) {\r\n        this.username = username;\r\n        this.password = password;\r\n        this.passwordCheck = passwordCheck;\r\n        this.nickName = nickName;\r\n        this.position = position;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/webmini2/src/main/java/com/sparta/webmini2/dto/SignupRequestDto.java b/webmini2/src/main/java/com/sparta/webmini2/dto/SignupRequestDto.java
--- a/webmini2/src/main/java/com/sparta/webmini2/dto/SignupRequestDto.java	(revision b33f662f2df8fa21b5d813deb57c69520cb257c6)
+++ b/webmini2/src/main/java/com/sparta/webmini2/dto/SignupRequestDto.java	(date 1649509541669)
@@ -1,4 +1,4 @@
-package com.sparta.webmini2.dto;
+package com.sparta.webmini2.Dto;
 
 import lombok.Getter;
 import lombok.Setter;
@@ -12,7 +12,7 @@
     private String nickName;
     private String position;
 
-    public SignupRequestDto(String username, String password, String passwordCheck, String nickName, String position) {
+    public SignupRequestDto(String username, String password, String passwordCheck,String nickName,String position) {
         this.username = username;
         this.password = password;
         this.passwordCheck = passwordCheck;
